src/lib/mina_graphql/types.ml

```diff
@@ -0,0 +1,3216 @@
+open Core
+open Async
+open Mina_base
+open Mina_transaction
+module Ledger = Mina_ledger.Ledger
+open Signature_lib
+open Currency
+open Schema
+
+include struct
```

Comment:
Is this done to narrow the scope of opening `Graphql_lib.Scalars`? If so, please explain this in a comment, because it looks confusing.

Reply:
Yes, I'll rewrite it slightly to remove the `include struct`

Reply:
See commit [a7b61d2](https://github.com/MinaProtocol/mina/pull/14037/commits/a7b61d2c9e312b3994a65e4f1e7efd2869d282a6)

Reply:
Thanks!